runTests.py

```diff
@@ -43,7 +43,8 @@ def runTests(configPath=None, useOldConfig=False, runTests=True):
     if runTests:
         testLoader = unittest.defaultTestLoader.discover(TEST_DIR)
         testRunner = unittest.TextTestRunner()
-        testRunner.run(testLoader)
+        return testRunner.run(testLoader)
```

Comment:
Ah. I just found the solution of `len(testRunner.run(testLoader).failures)`, but I think I like your answer better.